test/integration/challenger.js

```diff
@@ -80,24 +80,27 @@ describe('Challenger Client Integration Tests', function () {
 
       await timeout(3000)
 
-      for(i = 0; i < 11; i++) {
-        await timeout(5000)
-        const result = await getAllEvents(bridge.api.scryptVerifier, 'NewQuery')
-        result.length.should.be.gt(0)
-
-        let sessionId = result[0].args.sessionId.toNumber()
-        let _claimant = result[0].args.claimant
-        assert.equal(_claimant, claimant)
-
-        let session = await bridge.api.getSession(sessionId)
-        let step = session.medStep.toNumber()
-        // let highStep = session.highStep.toNumber()
-        // let lowStep = session.lowStep.toNumber()
-
-        let results = await bridge.api.getResult(session.input, step)
-
-        await bridge.api.respond(sessionId, step, results.stateHash, { from: claimant })
-      }
+      await new Promise(async (resolve, reject) => {
```

Comment:
this shouldn't be necessary

Reply:
The loop? You have to go back and forth between claimant and challenger

Reply:
The loop is fine, you don't need to wrap it in a promise like that. Why was this added?